ui/package.json

```diff
@@ -4,11 +4,11 @@
   "private": true,
   "dependencies": {
     "@fortawesome/fontawesome-free": "^5.7.2",
-    "@kubernetes/client-node": "git://github.com/scality/kubernetes-client-javascript.git#browser",
+    "@kubernetes/client-node": "git://github.com/carlito-scality/kubernetes-client-javascript.git#browser-dist",
```

Comment:
any reason we point to a fork ?

Reply:
it's is temporary because I do not have "write" rights to push my branch  to kubernetes-client-javascript. I'll made the change later

Reply:
I don't understand why you need that branch?

Reply:
@NicolasT I have problem with the "browser" branch which does not execute the "prepare" script to build the "dist" when "npm install". I do not understand why. So I create this new branch where I check "dist" in. It may be a temporary solution.